wagtail/admin/views/pages/move.py

```diff
@@ -106,5 +134,12 @@ def move_confirm(request, page_to_move_id, destination_id):
         {
             "page_to_move": page_to_move,
             "destination": destination,
+            "translations_to_move_count": len(
+                [
+                    page.id
+                    for page in pages_to_move
+                    if not page.alias_of and page.id != page_to_move.id
```

Comment:
I think the list comprehension will be fine.
nitpick: use `page.alias_of_id` so as to not trigger additional queries. Here and in #8002